### ISSUE TITLE
simplify `fish --version` to output only the version ID

### ISSUE THREAD

**Issue by krader1961** [fish-shell/fish-shell#4072]
Should `fish --version` should output just the version ID with no extraneous text? This came up in the context of issue #4068. I feel, as does @faho, that it should output just the version ID. But rather than unilaterally make that change I'm opening this issue to see if anyone can explain why making that change is a bad idea.

**Comment by faho** [comment]
Obviously I agree that the "--version" output of any program should only be the version number.

However, I'm not quite sure that changing it _now_ won't break anything. I haven't found anyone on github checking the `fish --version` output (I have only found someone checking that it produced output, not what in particular), and there are also other ways to get that information - `$FISH_VERSION` for example, which is the most likely way to get it from within fish.

**Comment by zanchey** [comment]
> Obviously I agree that the "--version" output of any program should only be the version number.

Virtually every program in my /bin and /usr/sbin disagrees :-)

**Comment by krader1961** [comment]
Wow! Like @zanchey I found on macOS nearly every command has verbose output for the `--version` flag with the GNU tools being the most verbose. And some, like `ksh --version`, being downright weird. Closing since it's easy enough to extract the version from the current message. And while changing the message is very unlikely to break anything and would make it marginally easier to test the version of the `fish` program it would be a more or less gratuitous change.
